Quoridor.Model/Graph.cs

```diff
@@ -1,88 +1,106 @@
         }
 
-        void makeAdjacencyList() {
-            adjacencyList = new LinkedList<int>[size];
+        void MakeAdjacencyList() 
+        {
+            _adjacencyList = new LinkedList<int>[_size];
 
-            for (int i = 0; i < adjacencyList.Length; ++i) {
-                adjacencyList[i] = new LinkedList<int>();
+            for (var i = 0; i < _adjacencyList.GetLength(0); ++i) 
+            {
+                _adjacencyList[i] = new LinkedList<int>();
             }
 
-            for (int i = 0; i < edges.GetLength(0); i++) {
-                addEdge(edges[i, 0], edges[i, 1]);
-                addEdge(edges[i, 1], edges[i, 0]);
+            for (var i = 0; i < _edges.GetLength(0); i++) 
+            {
+                AddEdge(_edges[i, 0], _edges[i, 1]);
+                AddEdge(_edges[i, 1], _edges[i, 0]);
             }
 
         }
 
-        void addEdge(int vertex1, int vertex2) {
-            adjacencyList[vertex1].AddLast(vertex2);
+        bool AddEdge(int vertex1, int vertex2) 
+        {
+            if (_adjacencyList[vertex1].AddLast(vertex2) != null) 
+                return true;
+            return false;
         }
 
-        bool removeEdge(int vertex1, int vertex2) {
-            return adjacencyList[vertex1].Remove(vertex2);
+        bool RemoveEdge(int vertex1, int vertex2) 
+        {
+            return _adjacencyList[vertex1].Remove(vertex2);
         }
 
-        public int[] getEdgesForVertex(int vertex) {
-            LinkedList<int> list = adjacencyList[vertex];
+        public int[] GetEdgesForVertex(int vertex) 
+        {
+            LinkedList<int> list = _adjacencyList[vertex];
             int[] edgs = new int[list.Count];
             list.CopyTo(edgs, 0);
             return edgs;
         }
 
-        public int[] dijkstraAlgorithm(int startVertex) {
-            int[] distances = new int[size];
-            bool[] used = new bool[size];
+        public int[] DijkstraAlgorithm(int startVertex) 
+        {
+            int[] distances = new int[_size];
+            bool[] used = new bool[_size];
 
-            for (int i = 0; i < size; i++) {
+            for (var i = 0; i < _size; i++) 
+            {
                 distances[i] = int.MaxValue;
             }
             distances[startVertex] = 0;
 
-            for (int i = 0; i < size; i++) {
+            for (var i = 0; i < _size; i++) 
+            {
                 int curr = -1;
-                for (int j = 0; j < size; j++)
+                for (var j = 0; j < _size; j++)
                     if (!used[j] && (curr == -1 || distances[j] < distances[curr]))
                         curr = j;
 
                 if (distances[curr] == int.MaxValue) break;
                 used[curr] = true;
 
-                LinkedList<int> list = adjacencyList[curr];
-                foreach (int vertex in list)
+                LinkedList<int> list = _adjacencyList[curr];
+                foreach (var vertex in list)
```

Comment:
I'm sorry if I confused you, but here we have to leave it the way it was.
[Link](https://docs.microsoft.com/ru-ru/dotnet/csharp/fundamentals/coding-style/coding-conventions#implicitly-typed-local-variables)
![image](https://user-images.githubusercontent.com/50269023/134670829-cd295473-c69b-4c17-8411-3961cfd082e0.png)